lib/src/intl_suggestors/intl_migrator.dart

```diff
@@ -40,7 +41,8 @@ class IntlMigrator extends ComponentUsageMigrator {
     super.migrateUsage(usage);
     final namePrefix =
         usage.node.thisOrAncestorOfType<ClassDeclaration>()?.name.name ??
-                usage.node.thisOrAncestorOfType<VariableDeclaration>()?.name.name ?? 'null';
+            usage.node.thisOrAncestorOfType<VariableDeclaration>()?.name.name ??
+            'null';
```

Comment:
note: this appears to be just a formatting change

Reply:
Yes, my IDE is set to auto-format, and it seemed to change a lot of stuff. Not clear why, this repo isn't set up for dart_dev, so it's just doing a generic dartfmt